index.html

```diff
@@ -440,30 +440,192 @@ <h1>Basic Concept</h1>
     </ul>
   </section>
 
+  <section class="normative">
+    <h1>CBOR Tags for CBOR-LD</h1>
+    <p>
+The first step in decoding a CBOR-LD payload is to recreate the term codec
+map that was used to encode it by processing the contexts in the payload. However,
+as these context URLs are encoded as integers, the consumer of the CBOR-LD
+must know what `Compression Table` (map from JSON-LD terms to integers) was used to
+compress the context URLs during creation to be able to reconstruct the term codec map. 
+The following sections define the exact mechanism by which this can be accomplished, allowing 
+an arbitrary CBOR-LD consumer to decompress any CBOR-LD payload meeting this specification.
+    </p>
+    <p>
+To this end, we have registered the range of CBOR tags 1500-1800** to be used for CBOR-LD,
```

Comment:
1500-1535 and 1792-1800 are reserved for other purposes and any future revisions that may/may not impact compression table lookups.

```suggestion
To this end, we have registered the range of CBOR tags 1536-1791** (0x0600-0x06FF) to be used for CBOR-LD,
```